### ISSUE TITLE
Documentation on change in symlink (npm link) behavior between v4 and v5

### ISSUE THREAD

**Issue by ORESoftware** [npm/npm#19636]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [x] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?

Is there documentation on change in symlink (npm link) behavior between v4 and v5? I have a package which succeeds with NPM v4.6.1, but fails on all NPM 5.x versions. The package uses `npm link` heavily so there is some sort of behavior change between the NPM major versions.

**Comment by tommck** [comment]
Yeah.. there are a bunch of differences and I'm slowly discovering them.  For instance, running "npm install" after you have done an "npm link" actually REMOVES the links, which is a BIG change
